### PR TITLE
changed button text to be more specific

### PR DESCRIPTION
**Pull request by adeola-ak**

### CODE REVIEW

File: client/web/src/enterprise/batches/list/NewBatchChangeButton.tsx [hunk 1]
```diff
@@ -7,6 +7,6 @@ interface NewBatchChangeButtonProps extends Pick<LinkProps, 'to'> {}
 
 export const NewBatchChangeButton: React.FunctionComponent<NewBatchChangeButtonProps> = ({ to }) => (
     <Link to={to} className="btn btn-primary">
-        <PlusIcon className="icon-inline" /> Create
+        <PlusIcon className="icon-inline" /> Create Batch Change
```
Comment (eseliger) [inline client/web/src/enterprise/batches/list/NewBatchChangeButton.tsx:10]: It should read "Create batch change", as the _entity_ of a batch change is lowercase, as per our definition. Only the _product_ Batch Changes is uppercase. :)

**Review by eseliger — COMMENTED** [review]
Thanks for the PR! Left one comment for a follow-up :)

Some meta on PR workflows:
Usually, we assign reviewers from the team (@sourcegraph/batchers in this case) to the PR, link to a ticket, if exists, that this will close (using the `Closes #ID` syntax), and defer merging until a PR has been approved by at least one person.

**Comment by eseliger** [comment]
Also, can you please check in on what [cla-bot posted](https://github.com/sourcegraph/sourcegraph/pull/28156#issuecomment-978343739) about your local git config? Otherwise, it'll always come back on every PR and nudge you to finally do it 😁

File: client/web/src/enterprise/batches/list/NewBatchChangeButton.tsx (hunk 1 above)
Comment (adeola-ak) [reply client/web/src/enterprise/batches/list/NewBatchChangeButton.tsx:10]: makes sense, thanks Erik!

**Pull request merged**